src/reference/asciidoc/metrics.adoc

```diff
@@ -150,7 +150,13 @@ registry.config().meterFilter(MeterFilter.deny(id ->
 
 Starting with version 6.0, Spring Integration utilizes a Micrometer Observation abstraction which can handle metrics as well as https://micrometer.io/docs/tracing[tracing] via appropriate `ObservationHandler` configuration.
 
-The observation handling is enabled on the `IntegrationManagement` components whenever an `ObservationRegistry` bean is present in the application context.
+The observation handling is enabled on the `IntegrationManagement` components whenever an `ObservationRegistry` bean is present in the application context and an `@EnableIntegrationManagement` is configured.
+To customize what set of components should be instrumented, an `observationPatterns()` attribute is exposed on the `@EnableIntegrationManagement` annotation.
+See its javadocs for a pattern matching algorithm.
+
+IMPORTANT: By default, none of the `IntegrationManagement` components are instrumented with an `ObservationRegistry` bean.
+Can be configured as `*` to match all components.
```

Comment:
```suggestion
Can be configured as `*` to match all components but, typically, patterns to match only inbound and outbound endpoints would be used, to instrument just those components.
```

Reply:
Why would we say that? 
You know I'm still going to add an observation to `MessageChannel` as well. I guess that one would be as a `PRODUCER` kind.
What objections do you have to not observe channels?
FYI in Spring Cloud Sleuth there was spans processing only on channels though...
Thanks

Reply:
It will kill performance if all the timers are switched from simple timers to observations.

Reply:
Right. I know what you mean about all those extra objects in messages processing, but I'm still going to expose an observation on channels like we do with timers at the moment.
Just to fulfill the picture and realign with whatever we have so far with SC Sleuth.
As we state now in docs: it is up end-user to decide what components to instrument.
Perhaps they will rely on an observation from Apache Kafka or Rabbit MQ on the input side and on the JDBC or HTTP on the outbound.
The Spring Integration will be configured only for channel.

Also: it doesn't matter where we instrument an observation, the garbage collection noise will be the same.

Therefore I don't see a reason in incorporate your suggestion to this sentence in the docs.